models/service_test.go

```diff
@@ -67,14 +67,42 @@ func TestServiceDetailParsing(t *testing.T) {
 				"namespace": "tutorial"},
 			Precedence: 1,
 			Route: map[string]map[string]string{
-				"labels": map[string]string{
```

Comment:
Upsie, I see there were some formatting things that didnt come up when I developed! :pray:

Reply:
Probably it was automatically from my IDE.